django/utils/translation/__init__.py

```diff
@@ -88,7 +88,7 @@ def ugettext_noop(message):
     return gettext_noop(message)
 
 
-def gettext(message):
+def gettext(message: str) -> str:
```

Comment:
I'm not sure this is accurate. The underlying `gettext()` function for when translations are active doesn't seem to make guarantees about type, and uses `mark_safe` in the case of messages that are already `SafeData`: https://github.com/django/django/blob/eb77e80de01e658541d4fcc3b0b38783ce4e6a7e/django/utils/translation/trans_real.py#L275